lib/libxdp/Makefile

```diff
@@ -131,7 +131,7 @@ $(TEMPLATED_SOURCES): %.c: %.c.in Makefile
 	$(QUIET_M4)$(M4) $(DEFINES) $< > $@ || ( ret=$$?; rm -f $@; exit $$ret )
 
 $(EMBEDDED_XDP_OBJS): %.embed.o: %.o
-	$(QUIET_GEN)$(LD) -r -b binary -o $@ -z noexecstack --format=binary $<
+	$(QUIET_CC)$(CC) -r -nostdlib -Wl,-z,noexecstack,--format=binary -o $@ $<
```

Comment:
With this change we're not using $LD at all anymore, so I think we should get rid of that from the configure script to avoid confusion...

Reply:
done